docs/customization.md

```diff
@@ -1,38 +1,53 @@
 ## Customization
-### Modifying the client library format
 
-Don't like the default client syntax?  Want a different language supported?  No problem!  OpenAPI Generator processes mustache templates with the [jmustache](https://github.com/samskivert/jmustache) engine.  You can modify our templates or make your own.
+### Modifying a template
 
-You can look at `modules/openapi-generator/src/main/resources/${your-language}` for examples. To make your own templates, create your own files and use the `-t` flag to specify your template folder.  It actually is that easy.
+Clone OpenAPI Generator and take a look at the following directory: `modules/openapi-generator/src/main/resources/${template}`. In here you'll see all of the generators available, for most programming languages, web application frameworks and web servers.
```

Comment:
The `${template}` should be changed back to `${your-language}` or something similar. The point of the previous text is that, for instance, if you're trying to modify `csharp`, you would look at `modules/openapi-generator/src/main/resources/csharp`. Changing this to `${template}` seems more confusing to me, and adding the following line that all generators exist under this directory would be incorrect.

Reply:
I thought this was clear enough, but I've expanded it: 

> Clone OpenAPI Generator and take a look at the following directory: `modules/openapi-generator/src/main/resources/${template}`. In here you'll see all of the generators available, for most programming languages, web application frameworks and web servers. For example, if you are looking for the C# template, it's named `csharp`. 

It's not just language, so saying `your-language` doesnt make any sense.